server/subscriptions.go

```diff
@@ -108,13 +108,26 @@ func (p *Plugin) SubscribeOrg(ctx context.Context, githubClient *github.Client,
 
 	listOrgOptions := github.RepositoryListByOrgOptions{
 		Type: "all",
+		ListOptions: github.ListOptions{
+			Page: 1,
+		},
 	}
-	repos, _, err := githubClient.Repositories.ListByOrg(ctx, org, &listOrgOptions)
-	if repos == nil || err != nil {
-		if err != nil {
-			mlog.Error(err.Error())
+	repos := []*github.Repository{}
+	fetchingRepos := true
+
+	for fetchingRepos {
```

Comment:
There is a idiomatic way to write this loop. Would you please take a look at https://github.com/google/go-github#pagination? I'm 3/5 that you should just follow the example there.